enspara/geometry/rotamer.py

```diff
@@ -14,6 +14,7 @@ def dihedral_angles(traj, dihedral_type):
     # transform so angles range from 0 to 360 instead of radians or -180 to 180
     angles = np.rad2deg(angles)
     angles[np.where(angles < 0)] += 360
+    angles[np.where(angles == 360)] = 0
```

Comment:
Just from a previous discussion, wanted to note that there may be cases where MDtraj won't return 360.0 exactly but rather something like like `359.9999998`, which could either 
1. Round up to 360, sneak past this `==` check, and cause an error,
2. Be caught by the `==` and be set to 0, which is actually going to cause that angle to be placed in the wrong basin.

I think the workaround idea we had was to add something like 
```angles[np.where(angles > 359.5)] = 359.5``` 
prior to this line, which would set all the angles not at 360 to 359.5 to ensure they get assigned to the correct rotameric basin.